.azure-pipelines/testbed_health_check.py

```diff
@@ -349,36 +374,74 @@ def check_interface_status_of_up_ports(self):
         logger.info("======================= check_interface_status_of_up_ports starts =======================")
 
         for sonichost in self.sonichosts:
+            if (self.is_multi_asic and
+                    self.duts_basic_facts[sonichost.hostname]["ansible_facts"]["dut_basic_facts"]["is_supervisor"]):
+                logger.info("Skip check_interface_status_of_up_ports on Supervisor.")
+                continue
```

Comment:
Can confirm that interface check is also skipped on Supervisor in sanity check (code [link](https://github.com/sonic-net/sonic-mgmt/blob/d299786c3220bf6103934e297a7827f189c0c755/tests/common/plugins/sanity_check/checks.py#L94))